packages/common/container-definitions/src/error.ts

```diff
@@ -3,10 +3,26 @@
  * Licensed under the MIT License.
  */
 
-import { ITelemetryProperties } from "@fluidframework/core-interfaces";
+import { FluidErrorType, IErrorBase } from "@fluidframework/core-interfaces";
 
 /**
- * Different error types the Container may report out to the Host
+ * Different error types the ClientSession may report out to the Host.
+ */
+export const ClientSessionErrorType = {
+	...FluidErrorType,
```

Comment:
@Josmithr @anthony-murphy is this an okay name for this error type? We can't use ContainerErrorType as it's already in use, but not sure whether to call this ClientErrorType or ClientSessionErrorType, so thoughts on naming would be appreciated!

Reply:
i would like to keep the naming in line with the package, so two ideas:
-  `ContainerErrorTypes`, if so, i would update it to `FluidErrorTypes` as well, and we can use the plural form for our const objects
- `LoaderType`, since this is the container loader package

I prefer ContainerErrorTypes

Reply:
I don't like client or session, as those both have a very specific meaning in the context of this layer